tests/DfE.FindInformationAcademiesTrusts.UnitTests/Pages/Trusts/Overview/TrustDetailsModelTests.cs

```diff
@@ -159,4 +150,21 @@ public async Task OnGetAsync_should_configure_TrustPageMetadata()
         _sut.TrustPageMetadata.PageName.Should().Be("Overview");
         _sut.TrustPageMetadata.TrustName.Should().Be("My Trust");
     }
+
+    [Fact]
+    public void SharepointLink_should_have_correct_default_value()
+    {
+        _sut.SharepointLink.Should().Be("");
```

Comment:
Is the correct default value empty string, if so lets rename the test as I'd assume this was a mistake